src/time_travel/datetime_patcher.py

```diff
@@ -10,39 +12,40 @@ class DatetimePatcher(object):
     For now patching only the @today function.
     """
     
-    def __init__(self, start_time=0):
+    def __init__(self, clock):
         """Create the patch.
         
         @start_time is time in seconds since the epoch.
         """
-        self.timestamp = start_time
+        self.clock = clock
         
-        self.real_datetime = datetime_lib.datetime.fromtimestamp
+        self.real_fromtimestamp = datetime_lib.datetime.fromtimestamp
         
         self.datetime = mock.Mock(wraps=datetime_lib.datetime)
         self.datetime.mock_add_spec(
             ["today", "now", "utcnow", "fromtimestamp"])
         
-        today_mock = mock.Mock(side_effect=self.get_time)
-        self.datetime.today = today_mock
+        self.datetime.today = mock.Mock(side_effect=self._now)
         
-        self.datetime_patcher = mock.patch('datetime.datetime', self.datetime)
-
-    def set_time(self, timestamp):
-        """Set the time returned by now functions.
+        self.patches = [mock.patch('datetime.datetime', self.datetime),
+                        ]
         
-        @timestamp - is time in seconds since the epoch.
-        """
-        self.timestamp = timestamp
-        
-    def get_time(self):
-        """Return a datetime object of the currently set time."""
-        return self.real_datetime(self.timestamp)
+    def _now(self):
+        return self.real_fromtimestamp(self.clock.get_timestamp())
     
+    def start(self):
+        """Start mocking datetime module."""
+        for p in self.patches:
+            p.start()
+            
+    def stop(self):
+        """Stop mocking datetime module."""
+        for p in self.patches:
+                p.stop()
+                
     def __enter__(self):
```

Comment:
Why do you need to allow this to be used as a context manager? I suggest you have a single API interface (=`TimeTravel`).